### PR TITLE
Reference existing documentation

### PR DESCRIPTION
**Pull request by Lyla-Fischer**
Updated the README to refer to existing documentation, including the XBlock tutorial, external XBlocks, and sample XBlocks

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Lyla-Fischer! I've created [OSPR-907](https://openedx.atlassian.net/browse/OSPR-907) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).
We can't start reviewing your pull request until you've   added yourself to the [AUTHORS](https://github.com/edx/xblock-sdk/blob/master/AUTHORS) file . Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by Lyla-Fischer** [comment]
I just noticed that I put one of my other pull requests as an ancestor of this pull request. It's a fairly uncontroversial pull request, though, and I can't figure out how to remove that particular commit from this pull request, so I'm hoping that we can just pretend like I opened this pull request after that uncontroversial one was accepted.

**Comment by sarina** [comment]
Interactive rebase will allow you to remove commits. See https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes - once you're in interactive rebase mode, just remove from the .txt file the commit(s) that you don't want present on your branch.

Alternatively, make a new branch and cherry pick only the commits you want onto it.

**Comment by Lyla-Fischer** [comment]
I just added some points to to my git-fu stats. :) Thanks!

**Comment by Lyla-Fischer** [comment]
I think that my updated git-fu dropped the comments and tags that were on previous commits. I'll re-add the comments.

File: sample_xblocks/README.rst [hunk 1]
```diff
@@ -4,19 +4,32 @@ Demo XBlocks
...
+
+  * **basic.slider:Slider** A block which presents a slider and remembers the student's setting
+
+  * **basic.view_counter:ViewCounter** 
+
+  * **basic.thumbs:ThumbsBlock** A block which shows the total number of times it has been viewed
+
+  * **basic.filethumbs:FileThumbsBlock** 
+
+Each of these blocks can be referenced in scenarios using a keyword in XML that is specified in the "entry_points" attribute of the setup tuple.
```
Comment (Lyla-Fischer) [inline sample_xblocks/README.rst]: @mhoeber My previous comment was dropped due to some rebasing, but I wanted to leave a record here that I asked you about proper documentation of these example classes. They could be useful as part of a tutorial, but quickly cause externally hosted documentation to become out of date as the examples mature.

**Comment by Lyla-Fischer** [comment]
Tagging @nedbat for opinions

**Comment by nedbat** [comment]
This is important, but it's going to be a few weeks before I can get to it.

File: README.rst [hunk 2]
```diff
@@ -84,9 +84,9 @@ as the default does.
 Making your own XBlock
 ----------------------
 
-Making an XBlock can be as simple as creating a Python class with a few
-specific methods.  The ``thumbs`` XBlock demonstrates an XBlock with state,
-views, and input handling.
+Making an XBlock involves creating a Python class that conforms to the XBlock 
+specification. See the sample_xblocks directory for examples and the 
+XBlock tutorial for a full walk-through: http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/index.html
```
Comment (nedbat) [inline README.rst]: Please use ReST syntax for making links with English link text, and a URL behind the scenes:
```
See the sample_xblocks directory for examples and `the XBlock tutorial`_ for a full walk-through.

.. _the XBlock tutorial: http://edx.readthedocs.org/projects/xblock-tutorial
```

File: README.rst [hunk 3]
```diff
@@ -110,13 +110,17 @@ own instance of Open edX, or released to the wider community).
 .. _XBlock review guidelines: https://openedx.atlassian.net/wiki/display/OPEN/XBlock+review+guidelines
 
 
-External XBlocks
+Example XBlocks
 ----------------
```
Comment (nedbat) [inline README.rst]: A minor nit, but please keep the punctuation line the same length as the section title. `</ocd>`

File: README.rst [hunk 4]
```diff
@@ -110,13 +110,17 @@ own instance of Open edX, or released to the wider community).
...
 
-We have been moving towards hosting XBlocks in external repositories, some of
-which have been installed and will appear in the workbench:
+Included in this repository are some example XBlocks that demonstrate how to use 
+various aspects of the XBlock SDK. You can see a more detailed description of 
+those examples in the readme located in that repository: https://github.com/edx/xblock-sdk/blob/master/sample_xblocks/README.rst
```
Comment (nedbat) [inline README.rst]: Another URL to rest-ify.

File: README.rst [hunk 5]
```diff
@@ -110,13 +110,17 @@ own instance of Open edX, or released to the wider community).
...
 
-ACID XBlock: https://github.com/edx/acid-block
+There is a rich community of XBlock developers that have put together a large 
+number of XBlocks that have been used in various contexts, mostly on the edx-platform. 
+You can see examples of what that community has done in the edx-platform wiki: 
+https://github.com/edx/edx-platform/wiki/List-of-XBlocks
```
Comment (nedbat) [inline README.rst]: Another URL to rest-ify.

File: sample_xblocks/README.rst [hunk 6]
```diff
@@ -4,19 +4,32 @@ Demo XBlocks
...
-  * **problem_demo:** A block which presents a problem and grades student responses
+  * **basic.structure:SidebarBlock** A variant of the vertical block
 
-  * **slider_demo:** A block which presents a slider and remembers the student's setting
+  * **basic.problem:ProblemBlock** A block which presents a problem and grades student responses
 
-  * **view_counter:** A block which shows the total number of times it has been viewed
+  * **basic.problem:TextInputBlock** 
 
+  * **basic.problem:EqualityCheckerBlock**
```
Comment (nedbat) [inline sample_xblocks/README.rst]: I'm wondering what to do with this block: it was a speculative design for how problems could be reimplemented as XBlocks, but we never used it, and likely never will convert capa to a new design like this.  I guess we should either remove it, or label it as "A possible design for how problems could be structured, but not used in edx-platform."

Comment (Lyla-Fischer) [inline sample_xblocks/README.rst]: I think that depends on whether we want to include examples of things that are not used in the edx-platform runtime. We don't currently have an alternative to show people how to build problems, however, I agree that the current level of exposure looks misleading to people using XBlock within edX's context.

**Comment by Lyla-Fischer** [comment]
There was some confusing stuff with what was pulled into this pull request when I rebased it most recently. I fixed it, and the pull request makes sense again.

**Comment by nedbat** [comment]
I've merged this, and will be making some further tweaks, coming to a PR near you.

**Pull request merged**